pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/DataObjectHelperController.php

```diff
@@ -183,7 +186,7 @@ public function importExportConfigAction(Request $request)
             $gridConfig = GridConfig::getById($gridConfigId);
             $user = $this->getUser();
             if ($gridConfig && $gridConfig->getOwnerId() != $user->getId()) {
-                $sharedGridConfigs = $this->getSharedGridColumnConfigs($this->getUser(), $gridConfig->getClassId());
+                $sharedGridConfigs = $this->getSharedGridColumnConfigs($this->getUser(), $gridConfig->getClassId()); // TODO parameter searchType is missing
```

Comment:
wanted behavior ... we wanna offer all configs (i.e. grid, variants, childs, ...), not just the grid config